Dockerfile

```
@@ -1,4 +1,4 @@
-FROM sonarsource/sonar-scanner-cli:5.0
+FROM sonarsource/sonar-scanner-cli:5.0.0
```

Switch the `USER` instead of updating the base image. 

```suggestion
FROM sonarsource/sonar-scanner-cli:5.0

USER root
```